sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/Subscribe.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.springsdk.annotations;
+
+import java.lang.annotation.*;
+
+
+public @interface Subscribe {
+
+  @Target(ElementType.METHOD)
+  @Retention(RetentionPolicy.RUNTIME)
+  @Documented
+  @interface ValueEntity {
+    Class<? extends kalix.javasdk.valueentity.ValueEntity<?>> value();
+  }
```

Comment:
Keep distinct annotations so we can enforce that the entity is either `ValueEntity` or `EventSourcedEntity`.